Src/VimCore/CommandUtil.fs

```diff
@@ -873,11 +874,18 @@ type internal CommandUtil
         _commonOperations.SetRegisterValue registerName RegisterOperation.Delete value
 
     member x.DisplayCharacterBytes () =
-        match SnapshotPointUtil.TryGetChar x.CaretPoint with
-        | None -> _commonOperations.Beep()
-        | Some c ->
-            let str = sprintf "%d" (int c)
-            _statusUtil.OnStatus str
+        if SnapshotPointUtil.IsEndPoint x.CaretPoint then 
+            _commonOperations.Beep()
+        else
+            let span = SnapshotCharacterSpan(x.CaretPoint)

```

Comment:
It makes me happy to see the character span type being put to good use!

Reply:
Indeed. I've found a few other uses for it, and it's underlying logic, in other features like `ga`.